packages/portfolio-contract/src/portfolio.flows.ts

```diff
@@ -83,34 +101,161 @@ export const openPortfolio = (async (
   ctx: {
     zoeTools: GuestInterface<ZoeTools>;
     makePortfolioKit: () => PortfolioKit;
+    contract: {
+      aavePool: string;
+      compound: string;
+      factory: string;
+    };
     inertSubscriber: GuestInterface<ResolvedPublicTopic<never>['subscriber']>;
   },
   seat: ZCFSeat,
-  _offerArgs: unknown, // TODO: USDN/USDC ratio
+  offerArgs: OfferArgsShapes, // TODO: USDN/USDC ratio
   // passed as a promise to alleviate contract start-up sync constraints
   localP: Promise<OrchestrationAccount<{ chainId: 'agoric-any' }>>,
 ) => {
   await null; // see https://github.com/Agoric/agoric-sdk/wiki/No-Nested-Await
   try {
-    const kit = ctx.makePortfolioKit();
+    const { makePortfolioKit, contract } = ctx;
+    const kit = makePortfolioKit();
+
+    const initRemoteEVMAccount = async (protocol: YieldProtocol) => {
+      const { evmChain } = offerArgs;
+      assert(evmChain, 'evmChain is required to open a remote EVM account');
+
+      const [agoric, axelar] = await Promise.all([
+        orch.getChain('agoric'),
+        orch.getChain('axelar'),
+      ]);
+
+      const { chainId, stakingTokens } = await axelar.getChainInfo();
+      assert.equal(stakingTokens.length, 1, 'axelar has 1 staking token');
+
+      const localAccount = await localP;
+      const caipChainId = PositionChain[offerArgs.evmChain];
+      const positionId = kit.keeper.add(protocol, caipChainId, localAccount);
+
+      try {
+        // @ts-expect-error
+        await localAccount.monitorTransfers(kit.tap);
+        trace('Monitoring transfers setup successfully');
+
+        const { give } = seat.getProposal();
+        const [[_kw, amt]] = Object.entries(give);
+
+        const assets = await agoric.getVBankAssetInfo();
+        const { denom } =
+          assets.find(a => a.brand === amt.brand) ||
+          (() => {
+            throw new Error(`${amt.brand} not registered in vbank`);
+          })();
+
+        await ctx.zoeTools.localTransfer(seat, localAccount, give);
+
+        const memo: AxelarGmpOutgoingMemo = {
+          destination_chain: evmChain,
+          destination_address: contract.factory,
+          payload: [],
+          type: AxelarGMPMessageType.ContractCall,
+          fee: {
+            amount: '1', // TODO: Get fee amount from api
+            recipient: gmpAddresses.AXELAR_GAS,
+          },
+        };
+
+        trace('Initiating IBC transfer');
+        await localAccount.transfer(
+          {
+            value: gmpAddresses.AXELAR_GMP,
+            encoding: 'bech32',
+            chainId,
+          },
+          {
+            denom,
+            value: amt.value,
+          },
+          { memo: JSON.stringify(memo) },
+        );
+
+        return positionId;
+      } catch (err) {
+        await ctx.zoeTools.withdrawToSeat(localAccount, seat, give);
+        const errorMsg = `EVM account creation failed ${err}`;
+        throw new Error(errorMsg);
+      }
```

Comment:
On my branch, we don't have coverage of this. I'm interested to see if we do on this branch.